src/prices/README.md

```diff
@@ -0,0 +1,179 @@
+# Price Oracle
```

Comment:
Please link to the original source of `src/prices` and its license.

Reply:
here is the source, it is an open source library provided by the messari subgraph team.

https://github.com/messari/subgraphs/tree/master/subgraphs/_reference_/src/prices

Reply:
Thanks. Please add a note to the README.md in the repository root.

Reply:
alright.